cpp/include/cudf/lists/list_view.cuh

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cuda_runtime.h>
```

Comment:
These includes are not needed by this header file.

Reply:
Also, this file does not need to be `.cuh`

Reply:
At the moment it doesn't need to be a .cuh, but as the list_view class fleshes out it will.

Reply:
Is it intended to be a device class then?  Currently we have `column`, `column_view`, `table`, `table_view` and even `strings_column_view` and `dictionary_column_view`. All of these are declared in `.hpp` files. Only the _device_ class counterparts are declared in `.cuh` header.

Should this be `list_device_view` instead?

Reply:
The intent is to be similar to string_view (__host__ __device__ constructor, comparison functions, etc).   Initially it had more code in it (check out this commit  https://github.com/rapidsai/cudf/pull/4972/commits/66ede6bbc6092db929054c64a767c57c5fb2919c#diff-fca3a371d5566ce31042844f02e16402) but it was skeletal so Jake suggested I just trim it down for now.    I would expect it to start getting fleshed out as more functionality is added.